SerializerManager.js

```diff
@@ -14,20 +14,15 @@ class Serializer {
 class SerializerManager {
   constructor (configuration) {
     this.serializers = configuration.serializers
-    this.cache = {}
   }
 
   findSerializer (types) {
-    const cacheValue = this.cache[types]
```

Comment:
how this is never used?

Reply:
Explained in a comment to the linked issue

Reply:
Thks, I re-read now

The cache is broken
https://github.com/fastify/fastify-accepts-serializer/issues/30